Backend/Tumultu/Application/FileVariants/Commands/CreateFileVariantCommand.cs

```diff
@@ -37,6 +31,13 @@ public async Task<Guid> Handle(CreateFileVariantCommand request, CancellationTok
             File = request.File
         };
 
+        var existingVariantsByUser = (await _repository.GetAllFileVariantsByFile(request.File))
```

Comment:
I am not sure if CreateFileVariant should be an independent command, since the user won't be able to trigger this action without uploading a file which is already implemented by CreateFileCommand. To be discussed I guess.

Reply:
So far every entity has CRUD created around it, I guess for temporary purposes it can stay like this. If we decide to restructure/delete it, lets do it with every command/query that needs it.